README.Rmd

```diff
@@ -87,6 +88,19 @@ Finally, you can calculate perceptual distances to a reference colour, as in
 distance(c("red","green","blue"), "red")
 ```
 
+### Minimal ggplot2 example
+```r
+library(magrittr)
+library(ggplot2)
+library(shades)
+data("mtcars")
```

Comment:
`magrittr` isn't needed here (see below), and I don't think the dataset needs to be loaded explicitly

Reply:
I included data("mtcars") to make it very obvious and clear for beginners. But now I agree with you, even ggplot's absolute beginner tutorials at tidyverse.org do not explicitly do it. I'll take it out! This is fun. Thanks for doing this with me, I'm inappropriately excited given how small this contribution is - it's just fun to learn how to contribute on gitHub!